src/primary_backup/node.rs

```diff
@@ -52,13 +64,16 @@ impl fmt::Display for Message {
 pub struct Node {
     pub state: State,
     pub store: Store,
+    pub cycle: usize,
```

Comment:
can you add doc comments explaining what view and cycle represent in this context?